pkg/build/controller/build/build_controller.go

```diff
@@ -874,8 +879,18 @@ func (bc *BuildController) createBuildPod(build *buildv1.Build) (*buildUpdate, e
 		buildutil.UpdateCustomImageEnv(build.Spec.Strategy.CustomStrategy, build.Spec.Strategy.CustomStrategy.From.Name)
 	}
 
+	// We need to know which CAs will be injected so that the build pod can reference each CA
+	// bundle by filename. This will ensure that the build pod is not scheduled until the
+	// referenced CAs are present in the CA ConfigMap.
+	buildCAData := bc.createBuildCAData()
+	caBundles := make([]string, len(buildCAData))
+	i := 0
+	for bundleName := range buildCAData {
+		caBundles[i] = bundleName
+		i++
+	}
```

Comment:
UPDATE: If a ConfigMap is mounted with `Items`, it will only mount the keys referenced in the `Items` list. We will be mounting the service-signing-cert by referencing it's key, since we are relying on the kubelet to hold off creating the container until the cert is injected. Therefore, we must also reference the additional trusted CA by key name.